remote-api/src/main/java/org/alfresco/rest/api/probes/ProbeEntityResource.java

```diff
@@ -177,33 +160,31 @@ else if (!result && (isLiveProbe || readySent)) // Are we sick
 
     private String getMessage(boolean isLiveProbe, boolean result, String message)
```

Comment:
This method removes a small amount of redundancy, but adds complexity to the logic elsewhere. I think it would be better to get rid of this method completely and just log the message desired.